File: 1/part_one.rb

max = 0
current = 0
File.readlines('input.txt').each do |line|
    line = line.strip()
    if line != ""
        current += line.to_i
    else
        if current > max
            max = current
        end
        current = 0
    end
end

puts max

File: 1/part_one.py
from collections import deque
# calories each elf is carrying is input

with open('input.txt', 'r') as file:
    most_calories = 0
    current_calories = 0
    for line in file:
        if line == "\n":
            # reset for new elf
            # most_calories = keep_highest(current_calories, most_calories)
            if current_calories > most_calories:
                most_calories = current_calories
            current_calories = 0
        else:
            current_calories += int(line)

    print(most_calories)